tests/test_transformations.py

```diff
@@ -0,0 +1,127 @@
+from pathlib import Path
+import math
+import random
+import os
+
+import pytest
+import numpy as np
+import cv2
+import imageio
+
+import segmentation_labeling_app.transforms.transformations as transformations
+
+test_frame = np.arange(25).reshape(5, 5)
+test_video = np.stack([test_frame]*2, axis=0)
+
+
+@pytest.fixture()
+def video_fixture():
+    np.random.seed(0)
+    random_video = np.array(
+        [[[5, 5], [5, 5]], [[2, 2], [2, 2]],
+         [[4, 4], [4, 4]], [[0, 0], [0, 0]],
+         [[3, 2], [10, 7]], [[11, 1], [0, 5]],
+         [[8, 2], [12, 1]], [[6, 3], [2, 8]],
+         [[11, 3], [1, 17]], [[8, 3], [9, 0]]]
+    )
+    yield random_video
+
+
+@pytest.mark.parametrize(("input_fps, output_fps, random_seed, strategy, "
```

Comment:
Please add cases for when input and output fps are not perfectly divisible

Reply:
done